### PR TITLE
Add waitlist support in RSVPs

### PR DESCRIPTION
**Pull request by francocorreasosa**
Adds `in_waitlist` column that will help implement a whitelist system as the one that meetup.com has right now.

### CODE REVIEW

**Comment by bernhard-hofmann** [comment]
White list or wait list? 😕

**Comment by francocorreasosa** [comment]
Small typo in the commit name, the code is using the correct wording 😄 sorry about that

**Comment by francocorreasosa** [comment]
Fixed, sorry again.

File: data/ddl.sql [hunk 1]
```diff
@@ -61,11 +61,15 @@ create table user_groups (
...
     user_id uuid references  users(id),
     event_id uuid references events(id),
+    date timestamptz not null,
+    in_waitlist boolean not null,
     primary key (user_id, event_id)
 );
 
+alter table rsvps alter column in_waitlist set default FALSE;
+
 create table user_bans (
```
Comment (dmmulroy) [inline data/ddl.sql]: Postgres enables inlining default values.  You can put this is inline in the create table statement: `in_waitlist boolean not null default false,`

Comment (dmmulroy) [reply data/ddl.sql]: I'd also want to make sure w/ @QuincyLarson that wait listing is an MVP feature. I suspect that's its going to be easy to start bikeshedding the database before we actually have the API hooked up and using it (see various open PRs).

**Comment by QuincyLarson** [comment]
@francocorreasosa I would say waitlists aren't MVP, like @dmmulroy pointed out. 

But then I'm sure people would immediately ask for this feature.

So yes - I think we can include them. I agree that `on_waitlist` sounds more natural.

**Comment by francocorreasosa** [comment]
Changed it, thanks

File: data/ddl.sql (hunk 1 above)
Comment (francocorreasosa) [reply data/ddl.sql]: done, thanks

**Comment by QuincyLarson** [comment]
@francocorreasosa Thanks again for creating this PR. 

And thanks to @dmmulroy and @vaibhavsingh97 for looking over this, too.

**Comment by nik-john** [comment]
@all-contributors please add @francocorreasosa to code

**Pull request merged**